ionoscloud/resource_dbaas_pgsql_cluster_test.go

```diff
@@ -97,6 +97,10 @@ func TestAccDBaaSPgSqlClusterBasic(t *testing.T) {
 				Config: testAccDataSourceDbaasPgSqlClusterBackups,
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttrPair(DataSource+"."+PsqlBackupsResource+"."+PsqlBackupsTest, "cluster_backups.0.cluster_id", DataSource+"."+PsqlBackupsResource+"."+PsqlBackupsTest, "cluster_id"),
+					resource.TestCheckResourceAttrSet(DataSource+"."+PsqlBackupsResource+"."+PsqlBackupsTest, "cluster_backups.0.size"),
+					resource.TestCheckResourceAttrSet(DataSource+"."+PsqlBackupsResource+"."+PsqlBackupsTest, "cluster_backups.0.location"),
+					resource.TestCheckResourceAttrSet(DataSource+"."+PsqlBackupsResource+"."+PsqlBackupsTest, "cluster_backups.0.version"),
+					resource.TestCheckResourceAttrSet(DataSource+"."+PsqlBackupsResource+"."+PsqlBackupsTest, "cluster_backups.0.is_active"),
 					utils.TestNotEmptySlice(PsqlBackupsResource, "cluster_backups.#"),
```

Comment:
A check for `earliest_recovery_target_time` is not present here because I had some problems with it. The tests were failing saying that the field didn't exist in the response. However, I tested locally and the field was there, not sure why this is happening, maybe it takes a little bit more time to compute this field. I will investigate more tomorrow.